### PR TITLE
DOCSP-29345 - Replace forEach

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-29345
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/docsp-29345-remove-foreach/

## Self-Review Checklist

- [X] Is this free of any warnings or errors in the RST?
- [X] Did you run a spell-check?
- [X] Did you run a grammar-check?
- [X] Are all the links working?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/limit.txt [hunk 1]
```diff
@@ -42,7 +42,7 @@ snippet to insert documents that describe books into the
      { "_id": 3, "name": "Atlas Shrugged", "author": "Rand", "length": 1088 },
      { "_id": 4, "name": "Infinite Jest", "author": "Wallace", "length": 1104 },
      { "_id": 5, "name": "Cryptonomicon", "author": "Stephenson", "length": 918 },
-     { "_id": 6, "name": "A Dance With Dragons", "author": "Tolkein", "length": 1104 },
+     { "_id": 6, "name": "A Dance With Dragons", "author": "Martin", "length": 1104 },
```
Comment (jordan-smith721) [inline source/fundamentals/crud/read-operations/limit.txt:45]: Reputation saved.

File: source/fundamentals/crud/read-operations/cursor.txt [hunk 2]
```diff
@@ -125,6 +95,23 @@ method to retrieve the subsequent element of the cursor:
    :start-after: start manual cursor example
    :end-before: end manual cursor example
 
+.. _node-fundamentals-cursor-array:
+
+Return an Array of All Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For use cases that require all documents matched by a query to be held
+in memory at the same time, use `toArray() <{+api+}/classes/FindCursor.html#toArray>`__.
```
Comment (jordan-smith721) [inline source/fundamentals/crud/read-operations/cursor.txt:104]: Nit/Suggestion: The rest of the sections on this page follow the structure "...use the X method...":

```suggestion
in memory at the same time, use the `toArray() <{+api+}/classes/FindCursor.html#toArray>`__ method.
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ some small suggestions

File: source/fundamentals/crud/read-operations/cursor.txt [hunk 3]
```diff
@@ -125,6 +95,23 @@ method to retrieve the subsequent element of the cursor:
...
+Note that large numbers of matched documents can cause performance issues
+or failures if the operation exceeds memory constraints. Consider using
+the ``for await...of`` syntax to iterate
+through results unless you want to return all documents at once.
```
[the remark this answers is not in the document]
Comment (mongoKart) [reply source/fundamentals/crud/read-operations/cursor.txt:108]: Ha! Good call.

**Pull request merged**